lib/iris/fileformats/pp.py

```diff
@@ -2197,6 +2194,11 @@ def save_pairs_from_cube(cube, field_coords=None, target=None):
     target : optional
         A filename or open file handle.
 
+    Yields
+    ------
+    :class:`iris.cube.Cube`, :class:`iris.fileformats.pp.PPField`.
+        A 2-dimensional slice of the input cube together with its associated pp-field.
```

Comment:
Can this be phrased so it's clear you get a stream of them ?
```suggestion
        2-dimensional slices of the input cube together with their associated pp-fields.
```

Would this read a bit better, to emphasize that there are a sequence of them ?